api/src/main/java/org/pmiops/workbench/reporting/ReportingMapper.java

```diff
@@ -3,28 +3,23 @@
 import java.util.Collection;
 import java.util.List;
 import org.mapstruct.Mapper;
-import org.mapstruct.Mapping;
-import org.pmiops.workbench.db.model.DbUser;
-import org.pmiops.workbench.db.model.DbWorkspace;
-import org.pmiops.workbench.model.ReportingResearcher;
-import org.pmiops.workbench.model.ReportingWorkspace;
+import org.pmiops.workbench.db.dao.projection.PrjUser;
+import org.pmiops.workbench.db.dao.projection.PrjWorkspace;
+import org.pmiops.workbench.db.model.DbStorageEnums;
+import org.pmiops.workbench.model.BqDtoUser;
+import org.pmiops.workbench.model.BqDtoWorkspace;
 import org.pmiops.workbench.utils.mappers.CommonMappers;
 import org.pmiops.workbench.utils.mappers.MapStructConfig;
 
 @Mapper(
     config = MapStructConfig.class,
-    uses = {CommonMappers.class})
+    uses = {CommonMappers.class, DbStorageEnums.class})
 public interface ReportingMapper {
-  @Mapping(source = "givenName", target = "firstName")
-  @Mapping(source = "userId", target = "researcherId")
-  @Mapping(source = "disabled", target = "isDisabled")
-  ReportingResearcher toModel(DbUser dbUser);
+  BqDtoUser toDto(PrjUser prjUser);
 
-  List<ReportingResearcher> toReportingResearcherList(Collection<DbUser> dbUsers);
+  List<BqDtoUser> toReportingUserList(Collection<PrjUser> users);
```

Comment:
Renamed researcher table to user.

Reply:
@calbach thoughts on a better naming convention here? I think technically a [data transfer object](https://en.wikipedia.org/wiki/Data_transfer_object) would be an aggregate of the whole payload, rather than an individual row. Just going back to `ReportingUser` for now.

Reply:
Reporting seems clear to me